tests/backup/backup_test.go

```diff
@@ -2661,6 +2661,265 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 	})
 })
 
+func GetScheduleUID(backupName string, orgID string, ctx context.Context) (string, error) {
+	backupDriver := Inst().Backup
+	bkpUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
```

Comment:
If this failed we should probably just return error here and not try to inspect anything with `bkpUid` that might not even be returned, because the `GetBackupUID()` returned `err` am I wrong? Could you please elaborate on this